pkg/app/app_list_test.go

```diff
@@ -243,6 +243,15 @@ test3                      	           	true   	true     	        	incubator/raw
 cache                      	my-app     	true   	true     	app:test	bitnami/redis   	17.0.7 
 database                   	my-app     	true   	true     	        	bitnami/postgres	11.6.22
 `,
-		})
+		}, cfg)
+	})
+}
+
+func TestListWithEnvironment(t *testing.T) {
```

Comment:
Can it be the same as the original as a function? `testListWithConfig`

Reply:
@yxxhero i've kept the name consistent with old name of test
it can be just TestList, or two TestList's with and without prepared charts

Reply:
@yxxhero so what should I do here?